index.html

```diff
@@ -1,18 +1,49 @@
-
+<!DOCTYPE html>
 <html>
     <head>
-       <h1>Manizha khorram </h1>
+       <title>Manizha khorram </title>
+       <meta name="Author" content="Manizha khorram">
+       <link rel="stylesheet" type="text/css" href="Style.css" />
     </head>
+    <header>
+        
+      
     
+     <nav>
+        <a href="#About/">About</a> |
+        <a href="#Experience/">Experience</a> |
+        <a href="#Connect/">Connect</a> |
+        </nav>
+
+        <h1 id="myName">Manizha khorram</h1>
+
+        
+    </header>
     <body>
-        <h2>About</h2>
-        <p>I am from Afghanistan and I love coding. I would like to work as professional sofware developer. I hold bachelor of computer science degree, I love problem solving and creating new things.I keen to learn challenging roles in javaScript, software development and related technologies.</p>
-        <h2>Experience</h2>
-        <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University Work Force Development Program) in Afghanistan.</p>
-        <h2>Connect</h2>
-        <a  href="mailto:manizh.khorram@gmail.com">Gmail</a> <br>
-        <a  href="https://github.com/Manizha-khorram">GitHub account</a> <br>
-        <a  href="https://www.linkedin.com/in/manizha-khorram/">LinkedIn account</a> 
+        <img src="Developer Girl.jpeg" alt="Picture of a girl which is coding" id="position">
+        <section>
+            <a href="#About"><h1>About</h1> </a>
+            <p>I am from Afghanistan and I love coding. I would like to work as professional sofware developer. I hold bachelor of computer science degree, I love problem solving and creating new things.I keen to learn challenging roles in javaScript, software development and related technologies.</p>
+        </section>
+
+        <section>
+            <a href="#Experience"><h1>Experience</h1></a>
```

Comment:
Just a side note on using more than one `<h1>` on a page:

> Using more than one `<h1>` is allowed by the HTML specification, but is not considered a best practice. Using only one `<h1>` is beneficial for screenreader users.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
I also found this article on "Link in Header? Or Header in Link?":
https://css-tricks.com/link-header-header-link/

:smile: